config/channels/in-memory-channel/300-in-memory-channel.yaml

```diff
@@ -96,3 +96,6 @@ spec:
                       replyURI:
                         type: string
                         minLength: 1
+                      deliveryURI:
+                        type: string
```

Comment:
maybe add a description here for this? We can do a follow-up, as others are also missing, but up to you :)

Reply:
I would prefer to do this a follow-up.